showcase/openfe_showcase.ipynb

```diff
@@ -2,12 +2,11 @@
  "cells": [
```

Comment:
<p>Yes the more I see this now, I think it makes more sense to mention the CLI at the end, and say that you can do all the steps above with these 3 CLI commands. I think it is easier to use the python API to talk about each step, since you can see what is going on with the python code, and its kinda satisfying (I think) for someone following along to hit the end and think "wow some of that python was over my head but I can do all that with a few CLI commands? awesome" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>